File: src/main/java/com/bezkoder/springjwt/models/Themes.java
package com.bezkoder.springjwt.models;

import com.fasterxml.jackson.annotation.JsonIgnore;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.SequenceGenerator;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Themes {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO, generator = "SEQUENCE")
	@SequenceGenerator(name = "SEQUENCE", sequenceName = "ID_THEME_SEQ")
	private Long IdTheme;
	
	private String titre;
	private Long grille;
	 
}

File: src/app/models/Themes.ts
import { Critere } from "./Critere"
import { GrilleEval } from "./GrilleEval"
import { Rhnom } from "./rhnom"

export class Themes {
    IdTheme!:number
    titre!:string
    grille!:number
    criteres: Critere[] = [];

}